### PR TITLE
remove unique constraint on tags and nodes_tags

### PR DESCRIPTION
**Pull request by vjeffrey**
### :nut_and_bolt: Description
adding a unique constraint breaks on boxes with real data so here i am removing that and reverting back to the original logic.
https://github.com/chef/automate/pull/263/commits

### CODE REVIEW

File: components/nodemanager-service/pgdb/migration/sql/02_add_unique_to_tags.up.sql [hunk 1]
```diff
@@ -1,5 +0,0 @@
-ALTER TABLE IF EXISTS tags
-  ADD UNIQUE (key, value);
-
-ALTER TABLE IF EXISTS nodes_tags
-  ADD UNIQUE (node_id, tag_id);
```
Comment (sdelano) [inline components/nodemanager-service/pgdb/migration/sql/02_add_unique_to_tags.up.sql]: I don't think we can just delete a migration that's been applied to all the nodes in our infrastructure can we?

I think we might need to:

1. edit the migration here to either not add the uniqueness constraint or do nothing
1. add another migration that if the table and constraint exist, delete them

This change was deployed already and deleting the code that deployed it won't un-deploy it.

Comment (vjeffrey) [reply components/nodemanager-service/pgdb/migration/sql/02_add_unique_to_tags.up.sql]: ya, i was gonna manually do that for our envs but it's probably safer to do it in the migration. ill push a file up

Comment (vjeffrey) [reply components/nodemanager-service/pgdb/migration/sql/02_add_unique_to_tags.up.sql]: (in case anyone other than us is consuming dev)

Comment (sdelano) [reply components/nodemanager-service/pgdb/migration/sql/02_add_unique_to_tags.up.sql]: We've also got about 15 deployed environments now. It's probably easier here.

File: components/nodemanager-service/pgdb/migration/sql/02_add_unique_to_tags.up.sql [hunk 2]
```diff
@@ -1,5 +0,0 @@
-ALTER TABLE IF EXISTS tags
```
Comment (sdelano) [inline components/nodemanager-service/pgdb/migration/sql/02_add_unique_to_tags.up.sql]: Probably not worth blocking over but it'd probably be nicer to have this just commented out with a comment explaining what happened.

**Comment by phiggins** [comment]
I know this is blocking things so there's some urgency involved, but we could also migrate the data to remove the duplicate things preventing the uniqueness constraints from being applied.

**Comment by vjeffrey** [comment]
> we could also migrate the data

ya, i started going down that path, but i'm unfamiliar with the process there so my attempts weren't going well. i opted for the quick fix.
are you familiar with deduping data in postgres? 
we could always try to do it in a next change

**Comment by phiggins** [comment]
Yeah, I'd be fine doing that later in another PR. 👍

**Pull request merged**